### ISSUE TITLE
The attachment

### ISSUE THREAD

**Issue by CodySwannGT** [gunnertech/thumbwar-ios-client#56]
I still dont know how to attach an image to a TW.  ALSO - the image should be "live" so that if you tap it you can see the image solely. The image sort of gets lost behind text and if the TW is about the image, you should have the ability to see the image solely.  Think Twitter.  (feature I know).

**Comment by smummolo** [comment]
agree. im not in love with the way the text and image are working together. I also dont like the formatting of the text on it's own at the moment. It doesn't look like the designs - any way to come closer / refine this element?

Re: text over images - I think we need to put a dark gradient over all images to bring up the text visually maybe like a 25% opacity black overlay on the images would work

**Comment by adamespn** [comment]
To answer the first question: The designs specify a large font but only show a very short amount of bet text. It's difficult to see what you imagine it ought to look like with much longer text or a smaller font. Can you throw up some examples?

I'm happy to put an overlay of 0x00000044 over all the war images. You can decide if that will work for you.

**Comment by smummolo** [comment]
im actually feeling like maybe we kill the image for now - the text looks great, images all look like hell. What do you think?

**Comment by CodySwannGT** [comment]
We can certainly make that happen.

**Comment by jiakeliu** [comment]
I think we decided to keep the image attachment because the content is more engaging. That said, it's still a friction to having to take a photo instead of using stock images. There's an iOS app, Polar, that implements a stock (Google?) image search and add feature in the content creation screen that could work well for us: https://itunes.apple.com/us/app/polar/id563322683?mt=8

Here's a video of the image search in action:
https://vine.co/v/OYTPFbIHqYT

I hope there's an API somewhere for this, since it'd be a pain to build it from scratch.

**Comment by adamespn** [comment]
It's okay if it's a pain. The question - in my mind - is whether or not its a core feature. I'm guessing it's not, based on my analysis of this app and the need it fills in the market. However, nobody has ever formally written a statement of what is considered core functionality and what is not.

**Comment by jiakeliu** [comment]
I do not think it's a core feature, but I just wanted to comment on this
ticket to capture the thought as I'm combing through the backlog.
